Source/Fuse.PushNotifications/Android/Impl.uno

```diff
@@ -296,13 +296,15 @@ namespace Fuse.PushNotifications
 		[Foreign(Language.Java)]
 		static void SpitOutNotification(Java.Object _listener, string title, string body, string bigTitle, string bigBody, string notificationStyle, string featuredImage, string sound, Java.Object _payload)
 		@{
+	        int requestID = (int) System.currentTimeMillis();
 			Context context = (Context)_listener;
 			Bundle payload = (Bundle)_payload;
 			Intent intent = new Intent(context, @(Activity.Package).@(Activity.Name).class);
-			intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+        	intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP);
```

Comment:
Please indent with tabs here...

Reply:
Hmm, [the documentation](https://developer.android.com/reference/android/content/Intent.html#FLAG_ACTIVITY_CLEAR_TOP) for `FLAG_ACTIVITY_CLEAR_TOP` says

> If set, and the activity being launched is already running in the current task, then instead of launching a new instance of that activity, all of the other activities on top of it will be closed and this Intent will be delivered to the (now on top) old activity as a new Intent.

...and [the documentation](https://developer.android.com/reference/android/content/Intent#FLAG_ACTIVITY_SINGLE_TOP) for `FLAG_ACTIVITY_SINGLE_TOP` says:

> If set, the activity will not be launched if it is already running at the top of the history stack.

...these two immediately sounds a bit like they are incompatible to me. However, further down on the documentation for `FLAG_ACTIVITY_CLEAR_TOP`, it seems to suggest they can be combined. :confused: 

@cbaggers: Does this make sense to you? I'm probably not smart enough to wrap my head around this right now ;)

Reply:
@kusma It is a bit odd it's one i'd like to test later this week when I have a free evening. 
@devadiab thanks for the PR! does this work without the change on this line? if not, what problem is caused and how did it manifest?